test/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source .env
+
+AWS_TEST_VPC_ID=$AWS_TEST_VPC_ID \
+AWS_TEST_SSH_KEYPAIR=$AWS_TEST_SSH_KEYPAIR \
+AWS_TEST_SUBNET_IDS=$AWS_TEST_SUBNET_IDS \
+AWS_TEST_AVAILABILITY_ZONES=$AWS_TEST_AVAILABILITY_ZONES \
+AZURE_TEST_SUBSCRIPTION_ID=$AZURE_TEST_SUBSCRIPTION_ID \
+go test -timeout 15m
```

Comment:
Is it necessary to explicitly pass all the env variables to `go test`? 🤔 I would expect them to be loaded into environment by the `source` cmd and then to be available to go with `os.Getenv`

Reply:
Yes, unfortunately, it is required to pass the environment variables explicitly. When using `source`, the imported environment variables are available in the very same process only. When invoking a sub-process (like `go` here), they aren't automatically passed.
Another approach can be to use `export` which will pass on environment variables also to any subsequent sub-processes, but therefore also giving less fine-grained control over exposing them.

Reply:
I see. yes, would not work without `export` as you stated.

One alternative could be to use joho/godotenv but likely this small convenient factor does not justify adding an additional library.